public/opensource-projects/index.json

```diff
@@ -0,0 +1,255 @@
+  },
+  "ui": {
+    "repo_url": "https://github.com/fedimint/ui",
+    "name": "ui",
+    "org": "fedimint",
+    "tags": ["lightning", "bitcoin", "p2p", "lightning-network", "e-cash", "gateways"],
+    "lang": ["Typescript"],
+    "issue": "https://github.com/fedimint/ui/issues"
+  },
+  "polar": {
+    "repo_url": "https://github.com/jamaljsr/polar",
+    "name": "polar",
+    "org": "",
+    "tags": ["lightning", "bitcoin", "lightning-network", "polar", "bitcoin-d", "c-lightning"],
+    "lang": ["Typescript", "Docker"],
+    "issue": "https://github.com/jamaljsr/polar/issues"
+  },
+  "bitcoin": {
+    "repo_url": "https://github.com/bitcoin/bitcoin",
+    "name": "bitcoin",
+    "org": "Bitcoin",
+    "tags": ["cryptography", "bitcoin", "p2p", "cryptocurrency", "blockchain", "protocol"],
+    "lang": ["C++", "Python", "C"],
+    "issue": "https://github.com/bitcoin/bitcoin/issues"
+  },
+  "xpub-converter": {
+    "repo_url": "https://github.com/jlopp/xpub-converter",
+    "name": "xpub-converter",
+    "org": "jlopp",
+    "tags": ["bitcoin"],
+    "lang": ["Javascript"],
+    "issue": "https://github.com/jlopp/xpub-converter/issues"
+  },
+  "bitcoinjs-lib": {
+    "repo_url": "https://github.com/bitcoinjs/bitcoinjs-lib",
+    "name": "bitcoinjs-lib",
+    "org": "bitcoinjs",
+    "tags": ["bitcoin", "bitcoinjs", "bitcoinjs-lib", "library"],
+    "lang": ["Typescript", "Javascript"],
+    "issue": "https://github.com/bitcoinjs/bitcoinjs-lib/issues"
+  },
+  "electrum": {
+    "repo_url": "https://github.com/spesmilo/electrum",
+    "name": "electrum",
+    "org": "spesmilo",
+    "tags": ["lightning", "bitcoin", "wallet", "bitcoin-wallet", "lightning-network"],
+    "lang": ["Python", "QML"],
+    "issue": "https://github.com/spesmilo/electrum/issues"
+  },
+  "bluewallet": {
+    "repo_url": "https://github.com/bluewallet/bluewallet",
+    "name": "bluewallet",
+    "org": "BlueWallet",
+    "tags": ["bitcoin", "blockchain", "cryptocurrency", "bitcoinjs", "electrum", "wallet"],
+    "lang": ["Javascript", "Typescript"],
+    "issue": "https://github.com/bluewallet/bluewallet/issues"
+  },
+  "LndHub": {
+    "repo_url": "https://github.com/BlueWallet/LndHub",
+    "name": "LndHub",
+    "org": "BlueWallet",
+    "tags": ["bitcoin", "lightning", "lightning-network"],
+    "lang": ["Javascript"],
+    "issue": "https://github.com/BlueWallet/LndHub/issues"
+  },
+  "rn-ldk": {
+    "repo_url": "https://github.com/BlueWallet/rn-ldk",
+    "name": "rn-ldk",
+    "org": "BlueWallet",
+    "tags": ["bitcoin", "lightning", "lightning-network", "react-native"],
+    "lang": ["C++", "Typescript"],
+    "issue": "https://github.com/BlueWallet/rn-ldk/issues"
+  },
+  "pheonix": {
+    "repo_url": "https://github.com/ACINQ/phoenix",
+    "name": "pheonix",
+    "org": "ACINQ",
+    "tags": ["lightning", "bitcoin", "lightning-network", "wallet", "self-custody", "bitcoin-wallet"],
+    "lang": ["C", "Kotlin"],
```

Comment:
We can add Swift as it's used about 12.9% in the code base.
https://github.com/search?q=repo%3AACINQ%2Fphoenix++language%3ASwift&type=code